src/core/Scene.ts

```diff
@@ -169,6 +169,8 @@ class Scene {
     const root = this._root;
     const view3D = this._view3D;
 
+    if (this._skybox) this._skybox.destroy();
```

Comment:
Optional chaining is also used elsewhere, so I think it can be used here as well.

Reply:
You're right, I should change that.